packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/call.js

```diff
@@ -186,6 +188,22 @@ describe(`plugin-phone`, function() {
               .then(() => assert.equal(call.status, `disconnected`))
           ]));
         });
+
+        it(`is not sending media`, () => {
```

Comment:
This is to prove that the camera isn't still sending after hangup, right? Can you make the description a little clearer about that goal?